mifosng-android/src/main/java/com/mifos/mifosxdroid/online/SurveyQuestionActivity.java

```diff
@@ -185,7 +191,7 @@ public void setSubtitleToolbar() {
             mToolbar.setSubtitle(getResources().getString(R.string.submit_survey));
         }
     }
-
+    
```

Comment:
please remove the indentation change in this line because it is not relevant to the issue that you are fixing.